src/LicenseConfiguration.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lendable\ComposerLicenseChecker;
+
+final class LicenseConfiguration
+{
+    /**
+     * @param list<string> $allowedLicenses
+     * @param list<string> $allowedPackagePatterns
+     */
+    public function __construct(public readonly array $allowedLicenses, public readonly array $allowedPackagePatterns)
+    {
+    }
+
+    public function allowsLicense(string $license): bool
```

Comment:
Yeah I thought about having `{'*' => ['MIT', 'BSD-3-Clause', 'foo/*' => ['BSD-2-Clause']}` style configuration, I think it would be a great next step?